src/js/components/search/visualizations/SubawardDropdown.jsx

```diff
@@ -0,0 +1,85 @@
+/**
+ * SubawardDropdown.jsx
+ * Created by Nick Torres 2/26/2024
+ */
+
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import { faFunnelDollar } from "@fortawesome/free-solid-svg-icons";
+import Analytics from 'helpers/analytics/Analytics';
+import NewPicker from "../../sharedComponents/dropdowns/NewPicker";
+
+const propTypes = {
+    size: PropTypes.string,
+    leftIcon: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
+    enabled: PropTypes.bool,
+    subaward: PropTypes.bool,
+    setSearchViewSubaward: PropTypes.func,
+    options: PropTypes.arrayOf(PropTypes.shape({
+        name: PropTypes.oneOfType([PropTypes.string, PropTypes.number]),
+        value: PropTypes.oneOfType([PropTypes.string, PropTypes.number])
+    })),
+    selectedValue: PropTypes.string
+};
+
+const SubawardDropdown = ({
+    selectedValue = 'prime',
+    options = [],
+    setSearchViewSubaward
+}) => {
+    const [selected, setSelected] = useState(selectedValue);
+    const dispatch = useDispatch();
+    const onClick = (e) => {
+        dispatch(setSearchViewSubaward(e === 'subaward'));
+        setSelected(e === 'prime' ? 'prime' : 'subaward');
+        if (e === 'subaward') {
+            Analytics.event({
+                event: 'search_subaward_dropdown',
+                category: 'Advanced Search - Search Fields',
+                action: 'Subawards Search',
+                gtm: true
+            });
+        }
+    };
+
+    const renderOptions = () =>
+        [
+            {
+                name: 'Prime Awards and Transactions',
+                value: 'prime',
+                onClick
+            },
+            {
+                name: 'Subawards',
+                value: 'subaward',
+                onClick
+            }
+        ];
+
+    // eslint-disable-next-line no-param-reassign
+    options = renderOptions();
```

Comment:
Do we need this render function?
Why not set options to this as a const?
Does options need to be a prop?
If it does need to be a prop because we might reuse this component, then this component should have a more general name.